tests/framework/utils.py

```diff
@@ -374,14 +374,14 @@ def _format_output_message(proc, stdout, stderr):
     return output_message
 
 
-def check_output(
-    cmd, ignore_return_code=False, no_shell=False, cwd=None, timeout=None
+def run_cmd(
+    cmd, check_output=False, no_shell=False, cwd=None, timeout=None
```

Comment:
Can this be `check`, same as `subprocess.run`? While we are at it, can we also use `shell` instead of `no_shell`?

Reply:
Done :)